Adds INP-related case studies (#10025)
GoogleChrome/web.dev: src/site/content/en/blog/economic-times-inp/index.md

---
layout: post

> title: Economic Times quest for fixing INP

Reviewer: Shame no image for this post.

Author: Yeah, but the prior Economic Times one we did last year didn't have a hero image, so I decided to just leave it for now.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/6cMvebBQKG2nhyc1eYeB.png", alt="A screenshot of the INP audit in CrUX. The INP for the page is 257 milliseconds, which is within the 'needs improvement' thresholds.", width="582", height="218" %}
</figure>

### INP CrUX trend

The traffic received on topic pages represents a significantly smaller portion of overall traffic. Hence, it was an ideal place for experimentation. The CrUX results along with the business outcomes were very encouraging, and led us to expand our efforts across the entire website to reap further benefits.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pkPEYIQIeEDKdIKOCFSY.png", alt="A screenshot of INP distributions as visualized in CrUX over a period of four months, starting in July 2022 and ending in October 2022. Values within the 'poor' and 'needs improvement' thresholds declined somewhat, while the values within the 'good' threshold increased.", width="800", height="190" %}
</figure>

### Akamai mPulse TBT Analysis

> We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution. Though TBT is a lab metric, we observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.

Reviewer: Confused about this sentence. If mPulse is RUM, and TBT is lab, how are you measuring TBT in mPulse?

Not big enough to hold up publishing but one to follow up afterwards.

Reviewer: Confirmed with some Akamai folks I'm with that it DOES measure TBT in the field.

Suggest slight tweak to the wording:

```suggestion
We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution, which measures TBT in the field. We observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.
```
